src/transformers/training_args.py

```diff
@@ -1054,7 +1055,7 @@ class TrainingArguments:
         default="length",
         metadata={"help": "Column name with precomputed lengths to use when grouping by length."},
     )
-    report_to: Optional[List[str]] = field(
+    report_to: Optional[Union[str, List[str]]] = field(
```

Comment:
This will make the argument `--report_to` unusable in the CLI, so it needs to be reverted. The type hints here are primarily here to help users in their IDE but for the `HfArgumentParser` to be able to properly get the arguments in the dataclass.

Reply:
Understood - I'll revert it!